docs/documentation/docs/controls/ProgressStepsIndicator.md

```diff
@@ -0,0 +1,61 @@
+# Progress Steps Indicator
+
+This control shows a progress of steps.
+
+Here is an example of the control in action:
+
+![ProgressStepsIndicator](../assets/ProgressStepsIndicator.png)
+
+ 
+
+## How to use this control in your solutions
+
+- Check that you installed the `@pnp/spfx-controls-react` dependency. Check out the [getting started](../../#getting-started) page for more information about installing the dependency.
+- In your component file, import the `Accordion` control as follows:
```

Comment:
Accordion should be ProgressStepsIndicator

Reply:
fixed